apps/engmanagement/src/pages/welcome/index.tsx

```diff
@@ -21,7 +21,9 @@ export const getServerSideProps: GetServerSideProps = async ({req, query}) => {
   let purchaseId = purchaseQueryParam
 
   if (session_id) {
-    const {stripeChargeId} = await stripeData(session_id as string)
+    const {stripeChargeId} = await stripeData({
+      checkoutSessionId: session_id as string,
+    })
```

Comment:
One of several examples of `stripeData` and `recordNewPurchase` switching to an `options` object for parameters to support passing in Stripe and Prisma `ctx`.